README.md

```diff
@@ -1,5 +1,24 @@
 # Northcoders News API
 
+https://alice-nc-news.herokuapp.com/
+
+This project is a RESTful API for accessing news articles and associated comments. Various HTTP requests can be performed to endpoints for articles, comments and users.
+
+# Working with the API
+
+This project can be forked and cloned from GitHub here: https://github.com/althrntn/Alice-BE-News-Project/tree/main
+
+Once cloned, several dependencies are needed to run the project: please install dotenv, express and postgres (pg). For testing purposes, jest, jest-sorted, jest-extended and supertest are also required.
+
+The API connects to two local databases - the development database and the testing database. Details of these can be found in the setup.sql file. The run-seed.js file in the db/seeds directory is set up to seed the development database when run.
+
+The API has been set up with a full testing suite found in the **tests** directory. These tests can be run using the 'test' script with 'npm run'. There is no need to seed the databases before running tests as the app.test.js file will re-seed the test databse prior to each test and close the database connection once testing is complete.
+
```

Comment:
Could maybe also have a bit on how to run it locally/not through tests i.e. run `npm start`